README.md

```diff
@@ -136,9 +136,9 @@ $ # install setup-envtest which configures etcd and kube-apiserver binaries for
 $ # https://book.kubebuilder.io/reference/envtest.html#configuring-envtest-for-integration-tests
 $ # https://github.com/kubernetes-sigs/controller-runtime/tree/master/tools/setup-envtest#envtest-binaries-manager
 $ go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
-$ # configure envtest to use k8s 1.22.x binaries
-$ setup-envtest use -p path 1.22.x
-$ source <(setup-envtest use -i -p env 1.22.x)
+$ # configure envtest to use k8s 1.24.x binaries
+$ setup-envtest use -p path 1.24.x
+$ source <(setup-envtest use -i -p env 1.24.x)
```

Comment:
Not a big deal, but I'd love it if this weren't so bash centric 🥺

Reply:
haha, I feel the same way. let's see what I can do in the following PR's